### PR TITLE
unit tests for OnlineJsonApis and OnlineRestApis

### LINKED ISSUE #182: Write Unit test Every for Api defined in OnlineRestApi and OnlineJsonApi

### PR DESCRIPTION
**Pull request by raystatic**
Fixes #182 
Changes: unit tests written for apis

### CODE REVIEW

File: onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt [hunk 1]
```diff
@@ -1,119 +1,104 @@
...
     }
 
     @Test
-    fun `GET NoteById `() {
-        val notes = jsonapi.getNotesByAttemptId("22842").execute().body()
-        notes?.let {
-            assertNotNull(it)
-        }
+    fun `POST createNote`(){
+
```
Comment (adityastic) [inline onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt]: Check create doubt as well

File: onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt [hunk 2]
```diff
@@ -1,119 +1,104 @@
...
+        assertNotNull(notes)
     }
 
     @Test
-    fun `GET NoteById `() {
-        val notes = jsonapi.getNotesByAttemptId("22842").execute().body()
-        notes?.let {
-            assertNotNull(it)
-        }
+    fun `POST createNote`(){
```
Comment (adityastic) [inline onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt]: Check create doubt as well

**Comment by aggarwalpulkit596** [comment]
Good Work @raystatic just resolve the conflict

File: onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt (hunk 2 above)
Comment (aggarwalpulkit596) [reply onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt]: it won't work for the trial courses

Comment (adityastic) [reply onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt]: Oh ok. Cool then

**Pull request merged**